### DISCUSSION TITLE
Suggestion to rename "cancelable" to "isCancelable"

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/evm-monorepo#213] [Ideas]
For consistency purposes, I suggest we rename to `cancelable` to `isCancelable`, to make it homologous to `isEntity`.

I also thought about the converse, i.e. renaming `isEntity` to `entity`, but this doesn't sound right to me.

**Comment by razgraf** [comment]
I like this boolean nomenclature where the binary flags start with `is[Condition]` or `are[Condition]`. It has been widely used in our interfaces code-base as well.

**Reply by PaulRBerg** [reply]
Cool, then it's settled. I created an issue to track this: https://github.com/sablierhq/v2-core/issues/214.
